template/build/core/phing/tasks/deploy.xml

```diff
@@ -99,11 +106,13 @@
 
     <!-- Copy required files from docroot. -->
     <copy todir="${deploy.dir}/docroot" overwrite="true">
-      <fileset dir="${docroot}">
+      <fileset dir="${docroot}" defaultexcludes="false">
```

Comment:
ACSF .gitignore, see https://www.phing.info/docs/guide/trunk/FileSet.html

Included files are 

```
Default excludes are: *~, #*#, .#*, %*%, CVS, CVS/**, .cvsignore, SCCS, SCCS/**, vssver.scc, .svn, .svn/**, ._*, .DS_Store, .darcs, .darcs/**, .git, .git/**, .gitattributes, .gitignore, .gitmodules
```

The cleanup task already removes .git directories.